gui/wxpython/gui_core/gselect.py

```diff
@@ -2103,8 +2104,35 @@ def GetDsn(self):
                 "PostgreSQL",
                 "PostGIS Raster driver",
             ):
+                ret = RunCommand("db.login", read=True, quiet=True, flags="p")
+                message = _(
+                    "PostgreSQL/PostGIS login was not setted."
+                    " Set it via <db.login> module, please."
+                )
+                if not ret:
+                    GError(parent=self, message=message)
+                    return
 
-                dsn = "PG:dbname=%s" % self.dbWidgets["choice"].GetStringSelection()
+                connection_string = None
+                for conn in ret.split(os.linesep):
```

Comment:
Any reason not to use splitlines?

```suggestion
                for conn in ret.splitlines():
```

Reply:
Ok, thanks.